src/mlir/verona-mlir.cc

```diff
@@ -14,107 +13,111 @@
 #include "mlir/InitAllDialects.h"
 #include "mlir/InitAllPasses.h"
 
+#include "llvm/Support/CommandLine.h"
 #include "llvm/Support/InitLLVM.h"
 
 namespace
 {
+  namespace cl = llvm::cl;
+  // Input file name
+  static cl::opt<std::string> inputFile(
```

Comment:
`static` is redundant (and confusing) for things in the anonymous namespace.

Reply:
sorry, copy and paste.